tests/unit/tls_test.cc

```diff
@@ -104,6 +104,8 @@ static future<> connect_to_ssl_addr(::shared_ptr<tls::certificate_credentials> c
     }).discard_result();
 }
 
+#ifdef SEASTAR_TESTING_WITH_NETWORKING
```

Comment:
Should be #if SEASTAR_TESTING_WITH_NETWORKING so it can be set to 0 or 1

Reply:
sorry, i don't quite follow here. why is `#if` better than `#ifdef` ? also, i am trying to follow the convention of other macros like `SEASTAR_DEBUG`, `SEASTAR_COROUTINES_ENABLED` and `SEASTAR_HAVE_DPDK`. i think this macro will not be a public interface used by developers directly, they should use `Seastar_TESTING_WITH_NETWORKING`.

Reply:
Personal preference, but (imho) `#if <boolean define>` is better that `#ifdef <symbol>` simply because it allows easier build script management (a -D<define>=<value> only has to change value, not remove/add def). It is also more concise (ask value, not existence)
Here we are using cmake, so maybe no a huge issue, but even cmake ensures the define is set to 1 for the proproc -> define configure things.

Reply:
okay. but we don't generate a header file using `configure_file()` yet.  anyway, i changed to `#if`. but i still think it'd be desirable to be more consistent with other macros.